newdle/client/src/util/timezones.js

```diff
@@ -0,0 +1,43 @@
+import preval from 'preval.macro';
+import 'moment-timezone';
+
+// This will just generate a pre-compiled list with all timezones and their
+// respective offsets.
+export const commonTimezones = preval`
+  require('moment-timezone');
+  const moment = require('moment');
+  const _ = require('lodash');
+
+  const tzData = require('moment-timezone/data/meta/latest.json');
+  const commonTzSet = Object.entries(tzData.countries).reduce((accum, [key, {zones}]) => {
+    zones.forEach(tz => {
+      accum[tz] = true;
+    })
+    return accum;
+  }, {});
+
+  // add some common timezones which are not official in any country
+  [
+    // UTC/GMT
+    'UTC',
+    'GMT',
```

Comment:
is GMT still relevant?

Reply:
I copied that [from here](https://github.com/stub42/pytz/blob/master/gen_tzinfo.py#L111).
Not sure whether everyone knows that UTC is.
https://www.timeanddate.com/time/gmt-utc-time.html

Maybe we should just remove both of them? Why would you want to hold a meeting in UTC?